tools/third_party.py

```diff
@@ -186,16 +186,12 @@ def get_buildtools_tool_path(tool):
                         tool + executable_suffix)
 
 
-# Download the given item from Google storage.
-def download_from_google_storage(item, bucket, base_dir):
+def download_from_google_storage2(sha1_file, bucket):
```

Comment:
What's the purpose of splitting up download_from_google_storage/download_from_google_storage2 ?

Reply:
I had to write a little script to download binaries that belonged to different platforms. Although I didn't commit the script, I think this split is helpful.

Reply:
IMO you could just add that to update_third_party.py